src/rebar_prv_compile.erl

```diff
@@ -71,9 +74,10 @@ build_app(State, AppInfo) ->
         end,
 
     %% Legacy hook support
```

Comment:
Technically this is no longer legacy since we run both hooks! Same comment exists in `clean`.